src/DependencyInjection/WebgriffeSyliusAkeneoExtension.php

```diff
@@ -43,6 +44,14 @@ final class WebgriffeSyliusAkeneoExtension extends AbstractResourceExtension imp
                 'sylius.repository.currency',
             ],
         ],
+        'channel_original_pricing' => [
+            'class' => ChannelOriginalPricingValueHandler::class,
+            'arguments' => [
+                'sylius.factory.channel_pricing',
+                'sylius.repository.channel',
+                'sylius.repository.currency',
+            ],
+        ],
```

Comment:
```suggestion
```

This public property is deprecated and should not be used anymore. This will also fix the roave BC check.